stories/topics/tech-note-aws-backup-restore.adoc

```diff
@@ -22,6 +22,16 @@
 * `<stack_name>-pulp_cert`
 * `<stack_name>-pulp_key`
 
+. The Ansible on Clouds Ops Container restore playbook fails when attempting to restore a secret starting with `-`. If the restore operation is failing, on your deployment please ensure the secret `<stack_name>-database_fields_symmetric_key` does not start with a `-`. Follow the steps below to check for this case and update the secret appropriately.
```

Comment:
If this is a known issue, it ought also to be in the Release Notes under 'Known issues'